JSDemos/Demos/Charts/Candlestick/description.md

```diff
@@ -1 +1,15 @@
+
+You can use the Candlestick type for an individual [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) or specify the type in the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/) object for all series in the Chart. Use the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/).[candlestick](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/CandleStickSeries/) object to configure properties of the Chart Candlestick series.
+
+In Candlestick series, each point consists of a rectangle (body) and a vertical line (shadow, wick or tail). The top and bottom values of a vertical line correspond to the highest and lowest prices of the stock. Use the [highValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#highValueField) and [lowValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#lowValueField) properties to specify data source fields for these prices.
+
+The bottom and topmost values of the point's body correspond to the opening and closing price of a stock. To specify these prices, use the [openValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#openValueField) and [closeValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#closeValueField) properties. If the stock closes higher than its opening price, no color fills the body and if the stock closes lower than its opening price, the Chart applies a color fill to the body.
+
+To reflect changes on the market, the Chart component compares the price of every point with the price of the previous point. The Chart paints the points whose price has decreased in a specific [reduction color](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/reduction/#color). To change color of other points, use the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[color](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/CandleStickSeries/#color) property; you can also specify the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[innerColor](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#innerColor) property to fill these points' bodies with color. To switch between high, low, open or close prices of points as sources for data comparison, use the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[reduction](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/reduction/).[level](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/reduction/#level) property.
+
+The following settings may help you make your chart more user-friendly:
```

Comment:
```suggestion
The following settings may help you make your chart more user-friendly:
```
So, something about this sentence is off for me. The idea is fine, but that word "may" bothers me. How about: "Apply the following settings to make your chart more user-friendly. " 
What do you think?

Reply:
I think that your sentence is better, I will replace it.